tracing/jaeger/src/main/java/io/helidon/tracing/jaeger/JaegerTracerBuilder.java

```diff
@@ -462,7 +510,12 @@ public Tracer build() {
             Resource serviceName = Resource.create(attributesBuilder.build());
             OpenTelemetry ot = OpenTelemetrySdk.builder()
                     .setTracerProvider(SdkTracerProvider.builder()
-                            .addSpanProcessor(SimpleSpanProcessor.create(exporter))
+                            .addSpanProcessor(BatchSpanProcessor.builder(exporter)
```

Comment:
It would be better to have a choice than a hardcoded option. Esp. for testing the `SimpleSpanProcessor` is better.
So maybe introduce `span-processor-type` that would be `simple` or `batch` (not sure if other supported).
Then use the one chosen (and we can have `batch` as default, if that is better for production)

Reply:
fixed